### PR TITLE
add first drivechain method

### PR DESCRIPTION
**Pull request by octobocto**
- **proto: add drivechain.proto**
- **scripts+makefile: add import sorting and rpc creating**
- **multi: run import sorting script**
- **commands: move additional commands into package**
- **multi/drivechain: add createsidechaindeposit command**

### CODE REVIEW

File: proto/bitcoin/drivechaind/v1alpha/drivechain.proto [hunk 1]
```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package bitcoin.drivechaind.v1alpha;
```
Comment (torkelrogstad) [inline proto/bitcoin/drivechaind/v1alpha/drivechain.proto:3]: Let's just go with `v1`, `v1alpha` is such a hassle to type all the time

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+SHELL=bash
+
+.PHONY: rpc
+
+PROTO_IN = $(shell find proto -name '*.proto')
+
```
Comment (torkelrogstad) [inline Makefile:6]: pls no Makefile, we've evolved to `just`

File: scripts/check-goimports.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    WRITE_ARG="-w"
+fi
+
+goimports -local $(go list -m | head -1) $WRITE_ARG -l $(git ls-files | \
+        grep --invert-match --file <(echo $deleted) |  \
+        grep --regexp '.*\.go$' | \
+        grep --invert-match '^openapi' |
+        grep --invert-match '^gen' |
+        grep --invert-match '^tests/mocks' |
+        grep --invert-match '^coinhandler/gen')
```
Comment (torkelrogstad) [inline scripts/check-goimports.sh:28]: oopsie wrong repo 😛

File: scripts/check-gogroup.sh [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+gogroup $WRITE_ARG \
+    -order=std -order=other -order=prefix=$(go list -m | head -1) \
+    $(git ls-files | \
+        grep --invert-match --file <(echo $deleted) |  \
+        grep --regexp '.*\.go$' | \
+        grep --invert-match '^openapi' |
+        grep --invert-match '^gen' |
+        grep --invert-match '^tests/mocks' |
+        grep --invert-match '^coinhandler/gen')
```
Comment (torkelrogstad) [inline scripts/check-gogroup.sh:31]: wrong repo

File: go.work [hunk 5]
```diff
@@ -0,0 +1,5 @@
+go 1.23.0
+
+use (
+	.
+)
```
Comment (torkelrogstad) [inline go.work:5]: Don't think we need this?

File: .github/workflows/build.yaml [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-go@v5
+        with:
+          go-version: "1.23"
+
```
Comment (torkelrogstad) [inline .github/workflows/build.yaml:13]: isn't there already a CI job that does this?

File: proto/bitcoin/drivechaind/v1alpha/drivechain.proto (hunk 1 above)
Comment (octobocto) [reply proto/bitcoin/drivechaind/v1alpha/drivechain.proto:3]: okidoki, also change the bitcoind package?

File: Makefile (hunk 2 above)
Comment (octobocto) [reply Makefile:6]: haha, oki, what the hell is just

File: .github/workflows/build.yaml (hunk 6 above)
Comment (octobocto) [reply .github/workflows/build.yaml:13]: not that I can find

File: proto/bitcoin/drivechaind/v1alpha/drivechain.proto (hunk 1 above)
Comment (torkelrogstad) [reply proto/bitcoin/drivechaind/v1alpha/drivechain.proto]: no, that's a breaking change i don't want to deal with

File: Makefile (hunk 2 above)
Comment (torkelrogstad) [reply Makefile]: https://github.com/casey/just

Comment (torkelrogstad) [reply Makefile]: Simple `make` alternative created by the ordinals guy

**Comment by octobocto** [comment]
It works! magix

**Pull request merged**